draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,88 @@ Appraisal Policy to make application-specific decisions such as authorization de
+The multi-chassis router provides higher throughput by interconnecting
+multiple routers and simpler management by logically seen as one router.
+Among these composing routers, there is only one main router that
+connects to the Network Management System (NMS) and the Verifier.
+Other routers are only connected to the main router by the network cables,
+and therefore they are managed and verified via this main router.
+So, in this case, the multi-chassis router is the Composite Attester,
+each router is an Attester and the main router is the Lead Attester.
+
+{{composite}} depicts the data that flows between the Composite Attester and Verifier for the remote attestation.
+
+~~~~
+                   .-----------------------------.
+                   |           Verifier          |
+                   '-----------------------------'
+                                 |  ^
+           Endorsements and      |  | Evidence of
+           Appraisal Policy for  |  | Composite
+           Evidence of Attesters |  | Attester
+                                 |  |
+.--------------------------------|--|-------------------------------.
+|                                v  |                               |
+|  .------------------------------------.                           |
+|  |  .-------------.                   |  Evidence of Attesters    |
+|  |  | Attesting   |-.                 | (via Internal Links or    |
+|  |  | Environment | |----+ Collecting |  Network Connections)     |
+|  |  '-------------' |    | Claims     |    /                      |
+|  |    '-------------'    |            |   / .------------.        |
+|  |                       v            |  /  |            |        |
+|  |                .-------------.     |<----| Attester B |--.     |
+|  |                | Target      |-.   |     '------------'  |     |
+|  |                | Environment | |   |<-------| Attester C |--.  |
+|  |                '-------------' |   |        '------------'  |  |
+|  |                  '-------------'   |<----------| ......     |  |
+|  | Lead Attester A                    |           '------------'  |
+|  '------------------------------------'                           |
+|                                                                   |
+|                       Composite Attester                          |
+'-------------------------------------------------------------------'
+~~~~
+{: #composite title="Conceptual Data Flow for Composite Attester"}
+
+In the Composite Attester, each Attester generates its own Evidence by its
+Attesting Environments collecting the Claims from its Target Environments.
+The Lead Attester collects the Evidence of all other Attesters and then
+generates the Evidence of the whole Composite Attester.
+
+After collecting the Evidence of other Attesters, the Lead Attester verifies the
+Evidence by using the Endorsements and Appraisal Policies, which are got from
```

Comment:
There is nothing elsewhere in the architecture about sending appraisal policies to an attester.
Similarly, there is nothing about a verifier sending endorsements to an attester.

Reply:
As above, this only makes sense, if there would be an optional Verifier role on the entity that is the also the "lead Attester". This "local Verifier" would be the consumer of Appraisal Policies. Again, there is now a lot of inconsistency between text and diagram. We should agree on how to proceed with the "local Verifier" scenario today (if my... deduction is actually correct).